readme.md

```diff
@@ -11,15 +12,14 @@ Automated dependency updates. Multi-platform and multi-language.
 
 ## Why Use Renovate?
 
-- Receive automated Pull Requests whenever dependencies need updating.
-- Define schedules to avoid unnecessary noise in projects (e.g. for weekends or outside of working hours, or weekly updates, etc)
-- Relevant package files are discovered automatically (e.g. supports
-  monorepo architecture such as lerna or yarn workspaces without further configuration)
-- Bot behaviour is extremely customisable via configuration files (config as code)
-- Use eslint-like shared config presets for ease of use and simplifying configuration
+- Receive automated Pull Requests whenever dependencies need updating
+- Define schedules to avoid unnecessary noise in projects (e.g. for weekends or outside of working hours, or weekly updates, etc.)
+- Relevant package files are discovered automatically (e.g. supports monorepo architecture such as Lerna or Yarn workspaces without further configuration)
+- Bot behavior is extremely customizable via configuration files (config as code)
+- Use ESLint-like shared config presets for ease of use and simplifying configuration
 - Lock files are natively supported and updated in the same commit, including immediately resolving conflicts whenever PRs are merged
-- Supports GitHub (.com and Enterprise), GitLab (.com and CE/EE), Bitbucket Server, Azure DevOps and Gitea. Bitbucket Cloud is also supported but considered in Beta until it gets the thumbs-up from more users
-- Open source (installable via npm/yarn or Docker Hub) so can be self-hosted or used via GitHub App
+- Supports GitHub (.com and Enterprise), GitLab (CE/EE), Bitbucket Server, Azure DevOps and Gitea. Bitbucket Cloud is also supported but considered in Beta until it gets the thumbs-up from more users
```

Comment:
Gitlab.com is supported but only by self-hosted renovate.

Reply:
Maybe we should be clearer on **how** we're supporting GitLab.com. I think there's a big difference between installing an app and having it work right away and having to manage/configure/host a bot via a GitLab pipeline.

Let me know what you think?

I'll undo the `.com` removal for GitLab right away. 👍